cocotb/share/lib/vpi/VpiCbHdl.cpp

```diff
@@ -379,6 +379,9 @@ int VpiSignalObjHdl::set_signal_value(s_vpi_value value_s,
 
     switch (action) {
         case GPI_DEPOSIT:
+#ifdef VERILATOR
```

Comment:
I think a comment here would be good, explaining why it's needed, even if 2.0 will remove it...